tests/crypto/read_file.c

```diff
@@ -121,7 +152,11 @@ oe_result_t read_dates(char* filename, oe_datetime_t* time)
     }
     buffer[len_date] = '\0';
 
+#ifdef _WIN32
```

Comment:
See comments on gmtime_r etc.  This should be able to use sscanf_s and on platforms where it's absent, a macro can map sscanf_s to sscanf.